infra/modules/providers/azure/storage-account/README.md

```diff
@@ -0,0 +1,118 @@
+# Azure Storage
+
+This Terraform based `storage-account` module grants templates the ability to configure and deploy cloud `storage containers` along with a `storage account` using Microsoft's _**Azure Storage**_ service.
+
+In addition, this module offers both "authentication" and "authorization" features:
+
+- For "authentication", this module automatically enrolls the deployed `storage account` into Microsoft's _**Managed Identities**_ service.
+
+- For "authorization", this module expects to receive the `object id` of any Azure Resource that will behave as a client of the `storage-account`. Enrollment as an Azure Resource client is achieved through Microsoft's _**Role Based Access Secruity**_ (RBAC) service.
+
+#### _More on Azure Storage_
+
+> "A storage account provides a unique namespace in Azure for your data. Every object that you store in Azure Storage has an address that includes your unique account name. A container organizes a set of blobs, similar to a directory in a file system. A storage account can include an unlimited number of containers, and a container can store an unlimited number of blobs." - Source: Microsoft's [Introduction to Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+
+This module deploys a `storage account` along with a `storage container` in order to satisfy blob storage scenarios which are optimized for storing massive amounts of unstructured data, such as text or binary data.
```

Comment:
Are we intending to support multiple containers in the account? Would this be achieved by specifying the resource you defined in this module multiple times with the same account name but different container names?

Reply:
After our discussion, we were able to achieve multiple containers with different names.
![multiple-containers](https://user-images.githubusercontent.com/10041279/59818641-50f8dc80-92ea-11e9-9de4-3407567657c2.jpg)

Reply:
Containers*

Reply:
Updated container names in screenshot:
![multiple-containers](https://user-images.githubusercontent.com/10041279/60022522-9a825800-9659-11e9-91e0-23ebf471969c.jpg)